src/assets/translations/en/main.json

```diff
@@ -1768,32 +1785,34 @@
       "juniorTranche": {
         "page-1": {
           "title": "Junior Tranche Strategy",
-          "body": "Junior tranches provide amplified optimized DeFi yields by carrying a higher grade of risks."
+          "body": "Enjoy boosted DeFi yields. Junior Tranches provide leveraged returns as they are compensated by their Senior counterparty for dragging more risk."
         },
         "page-2": {
           "title": "Junior Tranche strategies:"
         }
       },
       "monitorSystem": "Utilize Idle finance's monitoring system to find better-performing allocations and trigger a rebalance",
-      "offerHighReturn": "Offer high returns without the user constantly checking rates and burning gas on every transfer",
+      "offerHighReturn": "Offer maximized returns without the user constantly checking rates and burning gas on every transfer",
       "leverageYield": "Leverage yield using Senior tranches funds",
       "higherRisk": "Higher risk exposure by protecting senior tranches",
       "minRisk": "Minimized risk exposure",
       "builtInCoverage": "Built-in coverage for variable rate rewards",
       "noLockingPeriod": "No locking period or epochs",
-      "autoReinvest": "Auto-Reinvest accrued tokens"
+      "autoReinvest": "Auto-Reinvest accrued tokens",
+      "fundsCoveredJunior": "Funds covered by Junior Tranches",
+      "automation": "Automation and optimization of the yield generation across Aave, Compound, and multiple Senior Tranches"
     },
     "bestYield": {
       "title": "What is Best Yield?",
       "body": "Automatically maximize yield across multiple battle-tested DeFi protocols. Best-Yield strategy optimizes yield generation by socializing gas costs, automating yield generation by algorithmically shifting capital as new yield opportunities arise."
     },
```

Comment:
Should this go away in favor of Junior/Senior Best Yield as well?

Reply:
We are still using these in another place that isn't related to updating the copy here. So pull them out as we change stuff I think.

Reply:
Sounds good to me, let's get this in then